dashboard/app/views/devise/shared/_oauth_links.haml

```diff
@@ -27,3 +27,8 @@
 :javascript
 
   $('.oauth_sign_in').click(dashboard.clientState.reset);
+  // If the makerBridge is available, we are in the Maker App, which has it's
+  // own Google login button. Hiding this google-oauth button to avoid confusion.
+  if (!!window.MakerBridge) {
+    document.querySelector('.with-google_oauth2').style.display = 'none';
```

Comment:
Given that the code immediately above uses jquery, we could probably just jquery's `.hide()` here.